app/js/components/questions/new/NewQuestionPage.react.js

```diff
@@ -0,0 +1,110 @@
+import React, {PropTypes} from 'react'; // eslint-disable-line no-unused-vars
+import TagInputBox from './TagBoxes.react';
+import Sidebar from '../../layouts/Sidebar.react';
+import TinyMCE from 'react-tinymce';
+import tinymceConfig from '../../../config/tinymceConfig.js';
+
+const NewQuestionPage = ({
```

Comment:
It definitely is. I shouldn't have all the refractor fun alone now, should I? :)